source/bl_eeprom.c

```diff
@@ -10,6 +10,16 @@
 
 unsigned short crc16();
 
+void sw_reset(SW_RESET_REASON reason) {
```

Comment:
Seems like a bit of an odd place to have this function, but maybe it's worth the simplicity

Reply:
I put it here since this file gets downloaded to the obc software and it seemed good to have it accessible there